### ISSUE TITLE
Feature sorting

### ISSUE THREAD

**Issue by Vanbebar** [Microsoft/featuretimeline#89]
We understand that we can control the order in which features appear on the timeline via their position/order in the backlog - but this is only sustainable for a short period of time. As we close features they disappear from the backlog and thus lose control of their position/order within it. One symptom is when we choose to show closed features on the timeline. When we do so, closed features disrupt the current feature sorting by injecting themselves into the structure at weird locations and thus make the timeline difficult to use when showing stakeholders and the business what has been worked in the past.

In short, how can we control where a closed feature appears (vertically) in the timeline so that our timeline artifact can be presented in the most understandable manner?

Thank you.

**Comment by navneet-g** [comment]
The closed features still maintain their rank from the backlog, do you see otherwise? You can move closed features on your backlog, and the feature timeline will honor that order.

**Comment by navneet-g** [comment]
Closing due to no response.
